src/modules/imageresizer/ui/Models/ResizeOperation.cs

```diff
@@ -64,11 +64,25 @@ public void Execute()
 
                 foreach (var originalFrame in decoder.Frames)
                 {
+                    BitmapMetadata metadata = (BitmapMetadata)originalFrame.Metadata;
+                    if (metadata != null)
+                    {
+                        try
+                        {
+                            // Detect whether metadata can copied successfully
+                            _ = metadata.Clone();
+                        }
+                        catch (ArgumentException)
+                        {
+                            metadata = null;
```

Comment:
Yikes, that's a lot of information silently being deleted. I would sure hate to lose all my geotags because I selected **Resize the original pictures** and didn't know they got removed during the resize.

Reply:
Agreed, I would be upset to lose the metadata in my originals. In my own use I would never "resize originals" unless they were already copies of my true originals, but we can't assume that behaviour for everyone.

A couple of options that could help mitigate the data loss:
1. Detect the scenario where users are resizing the originals, and allow the error to occur. This enables users to avoid the error if they are creating copies, but avoid introducing an issue that would cause them to lose data in their originals. Could also add a warning at the end to make users aware that it happened.
2. Warn the user and give them the option to abort or continue with loss of metadata. Should include a "Do this for all images" option.

Reply:
This doesn't touch your original only the resize.

Reply:
Yes, this operation always creates a copy, but if you've selected to resize the originals (_settings.Replace == true), it will delete the original and replace it with the copy.  This happens at the end of this method (lines 102-107).